Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -1502,6 +1505,15 @@ - (void)showEditHistory {
     [self presentViewControllerEmbeddedInNavigationController:editHistoryVC];
 }
 
+- (void)showTalkPage {
+    SFSafariViewController *safariController = [[SFSafariViewController alloc] initWithURL:self.articleTalkPageURL];
+    [self.navigationController presentViewController:safariController animated:YES completion:nil];
+}
+
+- (NSURL *)articleTalkPageURL {
+    return [NSURL URLWithString:[self.articleURL.absoluteString stringByReplacingOccurrencesOfString:@"/wiki/" withString:@"/wiki/Talk:"]];
```

Comment:
A little surprised `Talk:` work in other langs too - they redirect appropriately :)

Reply:
I know, I was pleasantly surprised too :D

Reply:
`replace` works here, but maybe consider using `wmf_URLWithTitle:` and `wmf_title` instead:

     [self.articleURL wmf_URLWithTitle:[@"Talk:" stringByAppendingString:self.articleURL.wmf_title]]

nice cause it accesses the components directly